core/tests/gae_suite.py

```diff
@@ -70,14 +82,20 @@ def create_test_suites(
             top_level_dir=CURR_DIR
         )
     )
-
     return [master_test_suite]
 
 
 def main(args: Optional[List[str]] = None) -> None:
-    """Runs the tests."""
-    parsed_args = _PARSER.parse_args(args=args)
+    """Runs the tests.
+
+    Args:
+        args: list. A list of.
```

Comment:
I need help here. I can't figure out what `args` is a list of. I tried to print out it's contents but I was getting `None`.

Reply:
@deadex-ng see how the `args` variable is used in the script. That should give an idea why that variable is used.

Reply:
@gp201 I've figured it out and updated the docs.